README.md

```diff
@@ -26,29 +26,28 @@ See the [Raspberry Pi](README_RASPBERRY_PI.md) and [BeagleBone Black](README_BEA
 ### Platform support
 
 SuperCollider is tested with:
-- Windows 10 (32- and 64-bit) and MSVC 2019
-- macOS 12 and Xcode 13.4.1
+- Windows 10 64-bit and MSVC 2019
+- macOS 13 and Xcode 14.1
 - Ubuntu 22.04 and gcc 12
 
 SuperCollider is known to support these platforms:
-- Windows Vista, 7, 8, 10, 11
-- macOS 10.15-12.x
-- Ubuntu 20.04-22.04
+- Windows 10, 11
+- macOS 11-14
+- Ubuntu 22.04-24.04
 
-We also provide a legacy macOS binary for macOS 10.15 and above using Qt 5.9.
 
 SuperCollider has guaranteed support for:
 - Windows 10, 11
-- MSVC 2017, 2019
-- macOS 11, 12
-- Xcode 11-13
+- MSVC 2019, 2022
+- macOS 13, 14
+- Xcode 14-15
 - Debian >= 11
-- Ubuntu 20.04, 22.04
+- Ubuntu 22.04, 24.04
 - Fedora 36, 37
 - Arch Linux
 - gcc >= 9
 - clang >= 11
-- Qt >= 5.11
+- Qt >= 6.2
```

Comment:
I was thinking about not indicating support for Qt 5.15 aside from a note about compiling with Qt5 on macOS and the need to unlink qt6. Do you think it would be better to add a note here?

Reply:
Maybe we could add that it should work™️ but there is no support for it?

Reply:
This section says "guaranteed support for", so I don't know if I want to obfuscate it by something like 
> Qt >= 6.2 (it should still work with 5.15)

or

> Qt >= 6.2 (5.15 is still partially supported)

Any idea how to make this look clean, while providing that info?

Reply:
Maybe we can add a new line below, so at around line 52 add

> There is still limited support for Qt5. Further details are described in `README_MACOS.md`.

?